.github/workflows/fuzz.yml

```diff
@@ -0,0 +1,19 @@
+name: Fuzz
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install
+        run: |
+          python -m pip install -U pip
+          python -m pip install .
+        env:
+          CFLAGS: '-DDEBUG'
```

Comment:
One thing we might want to do in testing this is to change the initial size of the buffer in objToJSON to something much, much smaller. This will slow it down due to the repeated resizing of the buffer, but means we'll spend much more time near the limit of the buffer.